unic/ucd/normal/src/tables/canonical_combining_class_values.rsv

```diff
@@ -1,345 +1,345 @@
-// WARNING: Auto-generated by `tools/gen_ucd_tables.py`. DO NOT EDIT MANUALLY!
+// WARNING: Auto-generated by unic-gen. DO NOT EDIT MANUALLY!
```

Comment:
Oh, I didn't mean Rust vs. Python here at all. I just meant that keeping the `master` branch stable at any time is important, and this *stable* status also refers to the table generation tools generating the same file that's committed to the branch.

In this specific case, I'm not sure if we can/should get `unic-gen` in for the `0.5` release or not. (I love to see it happen, but don't want my time constraints delay the release more.)

Anyways, it's fine. The rest of the diff is good, and the Python fix is straightforward. We can land this and fix the script to go with it.